README.md

```diff
@@ -32,8 +32,16 @@
    ```bash
    sudo security add-trusted-cert -d -r trustRoot -k "/Library/Keychains/System.keychain" ./services/traefik/certs/output/programic-ca.crt
    ```
+   for Ubuntu:
+   ```bash
+   sudo cp ./services/traefik/certs/output/programic-ca.crt /usr/local/share/ca-certificates
+   sudo update-ca-certificates
+   ```
 2. Browser support:
-   * Chrome: out of the box
+   * Chrome: out of the box, except for Ubuntu. Chrome does not support system installed certificates anymore, 
```

Comment:
Ik begrijp deze zin niet helemaal. Ik denk dat het moet zijn:

``Chrome: out of the box, except for Ubuntu. Ubuntu does not support system installed certificates anymore,``